.gitignore

```diff
@@ -44,6 +44,7 @@ kubeconfig
 # Ignore output manifests
 config/samples
 manager_image_patch.yaml-e
+manager_pull_policy.yaml-e
```

Comment:
This is not ignored and was showing up in diffs.

Moreover, it would be nice to not manipulate `./config`. It feel yucky when our make file is injecting values into src which shouldn't be checked in.